package.json

```diff
@@ -8,6 +8,9 @@
     "engines": {
         "vscode": "^1.5.0"
     },
+    "extensionKind": [
+        "workspace"
```

Comment:
IS this 100% needed? I wonder if it will not cause additional issue for using remote with VS Code? 

The Scala syntax extension is pretty simple, so is there a drawback to having it enabled globally?

Reply:
Based on the documentation it would seem that what we want is to have `"ui"` first

https://code.visualstudio.com/api/advanced-topics/extension-host#preferred-extension-location says
> * "extensionKind": ["ui", "workspace"] — Indicates the extension prefers to run as a UI extension, but does not have any hard requirements on local assets, devices, or capabilities. When using VS Code, the extension will run in VS Code's local extension host if it exists locally and means the user does not have to install the extension on the remote. Otherwise, the extension will run in VS Code's workspace extension host if it exists there. When using VS Code for the Web with Codespaces, it will run in the remote extension host always (as no local extension host is available).

I believe we also need `"web"` to support https://github.dev/.

It seems that the default `"ui","workspace","web"` is what we need.

Reply:
According to the document, the difference between `ui,workspace,web` and `workspace` is about the preference, not the capability. In other word, do we want to enable this plugin for a non-Scala project?

If we don't sure about the preference, `workspace` is the default option, and `workspace` plugins can be on the local machine or on the remote machine or Codespace.

Reply:
The extension documentation seems to mention local vs remote workspaces, not workspaces that are language-specific. 

This plugin is only enabled on `.scala` files and not per workspace. Is there a use-case where someone would not want to highlight a `.scala` file with Scala highlighting?

We clearly want to run this on the local machine to avoid extra network traffic for something that does not require the remote machine and can be done locally.

Reply:
> This plugin is only enabled on .scala files and not per workspace. Is there a use-case where someone would not want to highlight a .scala file with Scala highlighting?

I think we should always highlight on Scala files, so especially that the extension works only on Scala files I don't think the change is needed.